lib/chef/knife/stalenodes.rb

```diff
@@ -34,65 +34,66 @@ class Stalenodes < Chef::Knife
+           long:        '--days DAYS',
+           description: 'The days since last check in',
+           default:     0
 
     option :hours,
-        :short        => "-H HOURS",
-        :long         => "--hours HOURS",
-        :description  => "Adds hours to days since last check in",
-        :default      => 1
+           short:       '-H HOURS',
+           long:        '--hours HOURS',
+           description: 'Adds hours to days since last check in',
+           default:     1
 
     option :minutes,
-        :short        => "-m MINUTES",
-        :long         => "--minutes MINUTES",
-        :description  => "Adds minutes to hours since last check in",
-        :default      => 0
+           short:       '-m MINUTES',
+           long:        '--minutes MINUTES',
+           description: 'Adds minutes to hours since last check in',
+           default:     0
 
     option :maxhost,
-        :short        => "-n HOSTS",
-        :long         => "--number HOSTS",
-        :description  => "Max number of hosts to search",
-        :default      => 2500
+           short:       '-n HOSTS',
+           long:        '--number HOSTS',
+           description: 'Max number of hosts to search',
+           default:     2500
 
     option :use_ec2,
-        :short        => "-e",
-        :long         => "--use-ec2",
-        :description  => "Indicate whether or not stale nodes are present in ec2",
-        :default      => false,
-        :proc => Proc.new { |key|
-          Chef::Config[:knife][:stalenodes] ||= {}
-          Chef::Config[:knife][:stalenodes][:use_ec2] = key
-        }
-
+           short:       '-e',
+           long:        '--use-ec2',
+           description: 'Indicate whether or not stale nodes are present in ec2',
+           default:     false,
+           proc: Proc.new do |key|
+             Chef::Config[:knife][:stalenodes] ||= {}
+             Chef::Config[:knife][:stalenodes][:use_ec2] = key
+           end
 
     def calculate_time
-      seconds = config[:days].to_i * 86400 + config[:hours].to_i * 3600 + config[:minutes].to_i * 60
+      seconds = config[:days].to_i * 86_400 +
+                config[:hours].to_i * 3_600 +
+                config[:minutes].to_i * 60
 
-      return Time.now.to_i - seconds
+      Time.now.to_i - seconds
     end
 
     def connection
       @connection ||= begin
         require 'fog'
-        connection = Fog::Compute.new(
-          :provider => 'AWS',
-          :aws_access_key_id => Chef::Config[:knife][:aws_access_key_id],
-          :aws_secret_access_key => Chef::Config[:knife][:aws_secret_access_key],
-          :region => Chef::Config[:knife][:region]
+        Fog::Compute.new(
+          provider: 'AWS',
+          aws_access_key_id: Chef::Config[:knife][:aws_access_key_id],
+          aws_secret_access_key: Chef::Config[:knife][:aws_secret_access_key],
+          region: Chef::Config[:knife][:region]
         )
       end
     end
 
-    def get_query
```

Comment:
Rubocop recommends against using `get_` in method names (it's considered redundant), and it made me realize that this method name was unclear.